lib/y2_installbase.pm

```diff
@@ -166,10 +166,19 @@ Performs steps needed to go from the "pattern selection" screen to
 
 sub go_to_search_packages {
     my ($self) = @_;
-    send_key 'alt-d';    # details button
-    assert_screen 'packages-manager-detail';
-    assert_and_click 'packages-search-tab';
-    assert_and_click 'packages-search-field-selected';
+    if (check_var('VIDEOMODE', 'text')) {
+        assert_screen 'patterns-list-selected';
+        wait_screen_change { send_key 'alt-f' };
+        for (1 .. 4) { send_key 'down'; }
+        send_key 'ret';
+        assert_screen 'pattern_selector';
+    }
```

Comment:
A bit curious about the needle. The function is ```go_to_search_packages```, but you assert pattern needles.